### DISCUSSION TITLE
Fill operation permanently breaks for the project when used too many times

### DISCUSSION THREAD

**Discussion by panglikesheep18** [Mojang/minecraft-editor#190] [Bugs]
Repro steps:
1: Have the project open, most preferably creating a new one for reproducing this issue
2: Find / select block(s) to fill
3: Fill the block(s) via Quick Fill / Fill Selection and repeat that 11 times at most

Expected result: Blocks you're filling is filled on even after tenth time

Observed result: After tenth (or earlier if you done that before on existing projects in the Game version listed below) attempt, the Editor will refuse to fill with "Fill operation failed with error: not a function" on history panel and yields an error everytime an action to the world is performed (such as brushing and summoning mobs), until the project is reloaded, Trying again without reloading the project yields an error on history panel with "Fill operation failed with error: Editor is still busy with a request. Wait for it to finish and try again.". Attempting to reload by leaving and re-entering does not fix the broken Fill operation.

Info:
Game version: 1.21.30.22
Frequency: Always

**Comment by PuppetMaster110** [comment]
I'm having the exact same problem

**Comment by panglikesheep18** [comment]
[Based of the changelog](https://github.com/Mojang/minecraft-editor/discussions/192),
"... Fixed a bug where Editor operations could exhaust all available ticking areas causing all future operations to fail ..."
Upon checking ticking areas on previous version after reproducing this issue, I have found that this bug could be reproduced because all available ticking areas were used up by the Fill operation

Though for some reason this wasn't fixed in 1.21.30.23 on Fill operation, you can workaround that by clearing all existing ticking areas by sending "/tickingarea remove_all" (plus exit and re-enter the project if you were on 1.21.30.22) and trying again.

**Comment by m-brisch** [comment]
Sorry about that. We have a fix in for the remaining issues that are present on `1.21.30.23` which should go out in the next preview.

**Reply by panglikesheep18** [reply]
But hey at least a workaround is discovered 👍

**Reply by panglikesheep18** [reply]
Because with the workaround, we can still use Fill operation before the issue was fixed.
Also without this particular changelog, I wouldn't even be able to discover a workaround.

**Comment by panglikesheep18** [comment]
Seemed fixed as of 1.21.30.24.
However in this version you have to repeat steps 2 and 3 instead of just step 3 as it would yield an error "Fill operation failed with error: Failed to commit transaction" if all of the blocks you're trying to fill in was the same as the block that has placed.
Anyways thanks for the dedication!
